tests/unit/backend/layers/common/base_test.py

```diff
@@ -283,6 +283,7 @@ def generate_dataset(
             )
         if artifacts is None:
             artifacts = [
+                DatasetArtifactUpdate(DatasetArtifactType.RAW_H5AD, f"s3://fake.bucket/{dataset_version_id}/raw.h5ad"),
```

Comment:
Evidently we weren't testing for the actual artifacts present in the fake bucket anywhere before, were we... 😏